### PR TITLE
DynamicStateDescriptionProvider: don't use deprecated method

### PR DESCRIPTION
**Pull request by lolodomo**
Signed-off-by: Laurent Garnier <lg.hc@free.fr>

### CODE REVIEW

**Comment by lolodomo** [comment]
@cweitkamp : this if for you ;)

**Comment by lolodomo** [comment]
I now moved 9 bindings to BaseDynamicStateDescriptionProvider.

File: bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/AVMFritzDynamicStateDescriptionProvider.java [hunk 1]
```diff
@@ -12,52 +12,30 @@
...
-import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
 
 /**
  * Dynamic provider of state options while leaving other state description fields as original.
  *
  * @author Christoph Weitkamp - Initial contribution
  */
-@Component(service = { DynamicStateDescriptionProvider.class, AVMFritzDynamicStateDescriptionProvider.class })
+@Component(service = { DynamicStateDescriptionProvider.class, BaseDynamicStateDescriptionProvider.class })
```
Comment (cweitkamp) [inline bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/AVMFritzDynamicStateDescriptionProvider.java]: Why did you change the component service declaration?

Comment (lolodomo) [reply bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/AVMFritzDynamicStateDescriptionProvider.java]: It was not intentional !

Comment (lolodomo) [reply bundles/org.openhab.binding.avmfritz/src/main/java/org/openhab/binding/avmfritz/internal/AVMFritzDynamicStateDescriptionProvider.java]: Fixed

**Comment by lolodomo** [comment]
Any chance to have a merge quickly ? I would like to work on an internationalization of the Sony Projector binding and I need this merge first.

**Comment by lolodomo** [comment]
Thank you
